polyfill/lib/civildatetime.js

```diff
@@ -0,0 +1,51 @@
+import { plus, pad, getMsSinceEpoch  } from './util.js';
+import { CivilDate } from './civildate.js';
+import { CivilTime } from './civiltime.js';
+import { Instant } from './instant.js';
+import { ZonedInstant } from './zonedinstant.js';
+
+const DATA = Symbol('data');
```

Comment:
Instead of only using a Symbol, what about using a `WeakMap` (with a Symbol → string fallback)? That way the state would be truly private when possible.